eZ/Publish/Core/settings/search_engines/shortcuts.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+    ezpublish.spi.search.content_handler.class: eZ\Publish\SPI\Search\Content\Handler
+    ezpublish.spi.search.location_handler.class: eZ\Publish\SPI\Search\Content\Location\Handler
+
+services:
+    ezpublish.spi.search.content_handler:
+        class: %ezpublish.spi.search.content_handler.class%
+        factory_service: ezpublish.spi.search
+        factory_method: contentSearchHandler
+
+    ezpublish.spi.search.location_handler:
+        class: %ezpublish.spi.search.location_handler.class%
+        factory_service: ezpublish.spi.search
+        factory_method: locationSearchHandler
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/b7fbbef6477ef26e2d238e7afb5c336bdc33fc0f